tools/build/workspace.bzl

```diff
@@ -145,8 +145,8 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
         locals = locals,
         organization = "KhronosGroup",
         project = "glslang",
-        commit = "8db9eccc0baf30c9d22c496ab28db0fe1e4e97c5",  # 8.13.3559
-        sha256 = "5c11a228d41ec011918b9c8beb60b6556745d30c8c856ec622beab5c5469152d",
+        commit = "3ee5f2f1d3316e228916788b300d786bb574d337",  # SDK-candidate-26-Jul-2020
```

Comment:
Let's go with https://github.com/KhronosGroup/glslang/releases/tag/10-11.0.0, which is marked as a stable release.

Reply:
Sure, done